### PR TITLE
Upgrade to React 18

### PR DESCRIPTION
**Pull request by alessandro308**

### CODE REVIEW

**Comment by kmiwa007** [comment]
hi @bcherny, any chance this PR can get merged soon?  Thanks.

File: package.json [hunk 1]
```diff
@@ -37,21 +37,21 @@
   "homepage": "https://github.com/coatue-oss/react2angular#readme",
   "peerDependencies": {
     "@types/angular": ">=1.5",
-    "@types/prop-types": ">=15",
-    "@types/react": ">=16",
-    "@types/react-dom": ">=16",
+    "@types/prop-types": ">=18",
```
Comment (kmiwa007) [inline package.json:40]: When trying to test install this I get
```
npm WARN Could not resolve dependency:
npm WARN peer @types/prop-types@">=18" from react2angular@7.0.1
npm WARN node_modules/react2angular
npm WARN   react2angular@"domotz/react2angular#react18" from the root project
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @types/prop-types@>=18.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```

So basically `@types/prop-types": ">=18"` doesn't exist.

**Comment by baileywickham** [comment]
@kmiwa007  It looks like this repo is unmaintained (last commit is 2019), is it worth forking and maintaining your own repo? I'm looking react 18 support too and it would be nice to have a maintained fork.

**Comment by alessandro308** [comment]
@baileywickham At the moment in production we use https://github.com/domotz/react2angular/tree/master with React 18 support. It's not something we update so frequently since it's pretty stable

**Pull request closed without merging**